js_task2/src/scss/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$font-secondary: rgb(129, 129, 129);
```

Comment:
use $color- at the beginning for this var to understand it is a color inside it

Reply:
done